server/database/data/exchangeMoney.js

```diff
@@ -0,0 +1,25 @@
+const exchangeMoney = require('../models/exchangeMoney');
+
+module.exports = async () => {
+  const newExchangeMoney = [
+    {
+      time: '2015-06-17T13:28:06.419Z',
+      ourRate: { buy: 3.46, sell: 3.47 },
```

Comment:
this is not enough information, so configure new way that handle currencies [USD, ILS , EUR .... ]